Vagrantfile

```diff
@@ -23,23 +23,26 @@ permanent = [
     :port => 10443,
     :dbport => 15432,
     :metricsport => 9113,
-    :autostart => true
+    :autostart => true,
+    :deploylatest => false
```

Comment:
Got it working with optional keys (check with machine.key?(keyname)) so it can still be declarative at the top of the file as to the nature of the machine, but don't need to include false values everywhere else. Will that do?

Reply:
LGTM